nailgun/entities.py

```diff
@@ -4612,7 +4612,7 @@ def __init__(self, server_config=None, **kwargs):
         super(Permission, self).__init__(server_config, **kwargs)
 
 
-class Ping(Entity, EntitySearchMixin):
+class Ping(Entity):
```

Comment:
This change will break at least one existing test: https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/endtoend/test_api_endtoend.py#L946
And leave it with no possible workarounds.

You're basically right, this entity is not searchable and the test is rather trying to read it, not search. But removing ability to send GET request completely isn't the best option either since it will introduce regression which can't be fixed.
How about adding `EntityReadMixin` and proper fields to entity so we start using it properly?